Makefile

```diff
@@ -27,6 +27,8 @@ deploy-appbundle:
 	cp \
 		$(BUILD_DIR)/app.bundle.min.js \
 		$(BUILD_DIR)/app.bundle.min.map \
+		$(BUILD_DIR)/do_external_connect.min.js\
+		$(BUILD_DIR)/do_external_connect.min.map\
```

Comment:
I'll let someone with more experience in that area comment. Thanks for explaining!